cairo-sys-rs/src/lib.rs

```diff
@@ -21,6 +21,7 @@ pub mod enums;
 
 use enums::{
     Status,
+    Content,
```

Comment:
Sorry, I missed that you already do full replace (`cairo_content_t` just have few use).
I like to explicitly comment `cairo_content_t` with indicating replacement, but as it not done in other cases we can ignore here too and use PR as is.
@GuillaumeGomez, what do you think?